addon/metrics-adapters/pendo.js

```diff
@@ -12,40 +12,36 @@ export default BaseAdapter.extend({
     let config = assign({}, this.config)
     let { apiKey } = config
 
-    assert(
-      `[ember-metrics] You must pass a valid \`apiKey\` to the ${this.toString()} adapter`,
-      apiKey
-    )
-
-    if (canUseDOM) {
-      /* eslint-disable */
-      (function(p,e,n,d,o){var v,w,x,y,z;o=p[d]=p[d]||{};o._q=[];
-      v=['initialize','identify','updateOptions','pageLoad'];for(w=0,x=v.length;w<x;++w)(function(m){
-      o[m]=o[m]||function(){o._q[m===v[0]?'unshift':'push']([m].concat([].slice.call(arguments,0)));};})(v[w]);
-      y=e.createElement(n);y.async=!0;y.src=`https://cdn.pendo.io/agent/static/${apiKey}/pendo.js`;
-      z=e.getElementsByTagName(n)[0];z.parentNode.insertBefore(y,z);})(window,document,'script','pendo');
-      /* eslint-enable */
+    // Workaround to make unit test pass, otherwise they will fail as assert will not be defined, cant find why
+    //  'import { assert } from "@ember/debug"' is not taken in account
+    if (typeof assert !== 'undefined') {
```

Comment:
So this is why test didn't passed in previous dependabot PR

Following this update, `assert` is not defined anymore when running unit test on `pendo` `metrics-adapters`

Checked a lot of topics in google but found nothing, can't understand why `import { assert } from "@ember/debug"` is not working in test environment.

Looking at how is it done in `ember-metrics`, like for the [Google Analytics metrics-adapter unit test](https://github.com/adopted-ember-addons/ember-metrics/blob/master/tests/unit/metrics-adapters/google-analytics-test.js#L28) there is nothing special ; also tested by replacing in our tests 
```
let adapter = Pendo.create({ config })
```
by 
```
let adapter = this.owner.factoryFor('ember-metrics-pendo@metrics-adapter:pendo').create({ config });
```
but we got same error 

---

I think it is an import issue in our tests but i clearly don't have the knowledge to fix this.

When trying to switch this file to native class as is it done in `ember-metrics`, had also import issues when adding the `classic` decorator (cf `import classic from 'ember-classic-decorator';`) it was failing to import it in unit test

If someone got an idea / understand why we have this import issue 🙏

Reply:
FFS it was a f* semi-colon! Damn it!